### ISSUE TITLE
custom loss with arguments 

### ISSUE THREAD

**Issue by dgiunchi** [keras-team/keras#8644]
Suppose I have a loss function that takes arguments which values can changes accordingly with inputs. Basically, my inputs are two images and an info file.  The info file is used only in the loss function but is loaded in the batch.
How can I do this? actually I see how to pass parameters like a normal function but since these values changes, and I load in generator and put like Inputs, I should embed them in output otherwise keras refuse to create the graph.
